subprojects/docs/src/docs/dsl/org.gradle.api.artifacts.ResolutionStrategy.xml

```diff
@@ -47,6 +47,9 @@
             <tr>
                 <td>dependencySubstitution</td>
             </tr>
+            <tr>
+                <td>getDisableGlobalDependencySubstitutionRules</td>
```

Comment:
This is not in line with the added property name

Reply:
Sorry for this. Fixed.

Reply:
No worry, thanks for being reactive 😉